### PR TITLE
OKTA-586867 : Authenticator icon primary color fix

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR changes the Authenticator icons to be themed with the primary dark color.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-586867](https://oktainc.atlassian.net/browse/OKTA-586867)

### Screenshot/Video:
Before: 
Screenshot 2023-03-29 at 10 28 06 AM

After: 
Screenshot 2023-03-29 at 10 27 38 AM

Before:
Screenshot 2023-03-29 at 10 16 32 AM

After:
Screenshot 2023-03-29 at 10 16 48 AM

### CODE REVIEW

File: src/v3/src/img/yubikeyDemo.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="43" height="58" fill="none" aria-labelledby="yubikeyIcon" role="img" viewBox="0 0 43 58">
     <!-- TODO: title needs to be i18n/l10n-friendly -->
     <title id="yubikeyIcon">yubikey</title>
-    <path fill="#00297A" fill-rule="evenodd" d="M10.61 0a9 9 0 0 0-9 9v.5a8.5 8.5 0 0 0 8.5 8.5v-2a6.5 6.5 0 0 1-6.5-6.5V9a7 7 0 0 1 7-7h13v14h-5.75v2h7.75v-3h8V3h-8V0h-15Zm15 5v8h6V5h-6Z" class="siwIconFillPrimary" clip-rule="evenodd"/>
+    <path fill="#00297A" fill-rule="evenodd" d="M10.61 0a9 9 0 0 0-9 9v.5a8.5 8.5 0 0 0 8.5 8.5v-2a6.5 6.5 0 0 1-6.5-6.5V9a7 7 0 0 1 7-7h13v14h-5.75v2h7.75v-3h8V3h-8V0h-15Zm15 5v8h6V5h-6Z" class="siwIconFillPrimaryDark" clip-rule="evenodd"/>
```
Comment (glenfannin-okta) [inline src/v3/src/img/yubikeyDemo.svg:4]: cc @alexdahl-okta @narayun-okta Should we be applying the dark theme color on just auth coins or all images?

**Review by glenfannin-okta — COMMENTED** [review]
Don't forget the test in `test/e2e/tests/theming.ts`, there is one there that asserts on the color of the auth coin that will require fixing.

File: src/v3/src/img/yubikeyDemo.svg (hunk 1 above)
Comment (leemchale-okta) [reply src/v3/src/img/yubikeyDemo.svg:4]: i believe alex said the 2 color imgs use the dark theme color so that was why I added it here aswell

**Review by shawyu-okta — APPROVED** [review]
Do you mind adding a few screenshots of some images (definitely don't need all of them) but just to illustrate the difference from how it looked before?

Address @glenfannin-okta comment about the test and get confirmation from @alexdahl-okta and should be good to go

**Comment by leemchale-okta** [comment]
> 

ah thought I already put up the screen shots but looks like i forgot.  Will do thank you!

**Pull request merged**